docs/api.rst

```diff
@@ -444,12 +444,12 @@ Querying general network events
 
 
 You can query for non-channel specific events by making a ``GET`` request to the
-endpoint of the token registry contract. ``GET /api/<version>/events/network/<token_registry_address>``
+following endpoint. ``GET /api/<version>/events/network``
```

Comment:
That is understood. I want to be clear that I don't agree that because the Raiden node will have a single registry the API doesn't need to be future proof.